code/modules/mob/living/carbon/xenomorph/abilities/ability_helper_procs.dm

```diff
@@ -172,7 +172,7 @@
 		T.update_xeno_hostile_hud()
 	to_chat(H, SPAN_XENOHIGHDANGER("You can move again!"))
 
-/proc/xeno_throw_human(mob/living/carbon/H, mob/living/carbon/Xenomorph/X, direction, distance)
+/proc/xeno_throw_human(mob/living/carbon/H, mob/living/carbon/Xenomorph/X, direction, distance, var/shake_camera = TRUE)
```

Comment:
you don't need var here, the rest don't even have it, Consistency

Reply:
youre a nerd